news-bundle/src/EventListener/GeneratePageListener.php

```diff
@@ -43,21 +41,21 @@ public function __invoke(PageModel $pageModel, LayoutModel $layoutModel): void
 
         $this->framework->initialize();
 
-        $adapter = $this->framework->getAdapter(NewsFeedModel::class);
+        $adapter = $this->framework->getAdapter(PageModel::class);
 
-        if (!($feeds = $adapter->findByIds($newsfeeds)) instanceof Collection) {
+        if (!($feeds = $adapter->findMultipleByIds($newsfeeds)) instanceof Collection) {
```

Comment:
should we make sure these are feed pages?

Reply:
Fixed in 97f8d3dff